Server/index.js

```diff
@@ -1,28 +1,32 @@
-const express = require('express')
-const helmet = require('helmet')
-const cors = require('cors')
-const authRouter = require('./routes/authRoute')
-const { connectDbAndRunServer } = require('./configs/db')
-require('dotenv').config()
+const express = require("express");
+const helmet = require("helmet");
+const cors = require("cors");
+const authRouter = require("./routes/authRoute");
+const monitorRouter = require("./routes/monitorRoute");
+const { connectDbAndRunServer } = require("./configs/db");
+require("dotenv").config();
```

Comment:
Double quotes and semi-colons 😄 Looks great!

Reply:
I think my linter is just set up to do that on save :laughing:  Happy to use whatever our team standard is!